### PR TITLE
Rhs/cobra

### PR DESCRIPTION
**Pull request by rhs**

### CODE REVIEW

File: cmd/kubeapply/main.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	case "no":
+		return false
+	case "0":
+		return false
+	case "":
+		return false
+	}
+
+	return true
+}
```
Comment (LukeShu) [inline cmd/kubeapply/main.go]: Is there a reason you didn't use `strconv.ParseBool()`?

File: cmd/kubeapply/main.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+func adapt(f func(*cobra.Command, []string) error) func(*cobra.Command, []string) {
+	return func(cmd *cobra.Command, args []string) {
+		err := f(cmd, args)
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+	}
+}
```
Comment (LukeShu) [inline cmd/kubeapply/main.go]: `adapt()` is a weird idiom.  What about setting `cmd.SilenceUsage = true` from `PersistenPreRun`? Or even just at the beginning of your `RunE` function?

Comment (rhs) [reply cmd/kubeapply/main.go]: I'm not sure I understand the question. What does SilenceUsage do for me?

File: cmd/kubeapply/main.go (hunk 1 above)
Comment (rhs) [reply cmd/kubeapply/main.go]: No clue, that code is from ages ago. Maybe I didn't know it existed. Thanks for the pointer. ;-)

**Pull request merged**